nilearn/maskers/tests/test_surface_labels_masker.py

```diff
@@ -6,9 +6,22 @@
 
 from nilearn._utils.class_inspect import check_estimator
 from nilearn._utils.helpers import is_matplotlib_installed
-from nilearn.maskers import SurfaceLabelsMasker, SurfaceMasker
+from nilearn.maskers import SurfaceLabelsMasker
 from nilearn.surface import SurfaceImage
 
+
+@pytest.fixture
+def _sklearn_surf_label_img(surf_mesh):
```

Comment:
Had to create a new label img fixture locally because an invalid scikit-learn check was passing for unrelated reasons...